packages/locale/src/translations/de.json

```diff
@@ -438,7 +438,9 @@
                 "birth_date": "Geburtsdatum",
                 "errors": {
                     "isRequired": "{fieldName} ist erforderlich",
-                    "isInvalid": "{fieldName} ist ungültig"
+                    "isInvalid": "{fieldName} ist ungültig",
+                    "paymentSourceInfoCannotBeVerified": "Die Kombination aus Ihrem Namen und Ihrer Anschrift konnte nicht validiert werden. Bitte korrigieren Sie Ihre Daten und versuchen Sie es erneut. Weitere Informationen finden Sie in den Ratepay <a href='https://www.ratepay.com/legal-payment-dataprivacy/?lang=de' target='blank'>Datenschutzbestimmungen</a> oder nutzen Sie das Ratepay <a href='https://www.ratepay.com/kontakt/' target='blank'>Kontaktformular.</a>",
+                    "paymentSourceDeclinedByProcessor": "Die gewählte Zahlungsart kann nicht genutzt werden. Diese Entscheidung basiert auf einem automatisierten Datenverarbeitungsverfahren. Weitere Informationen finden Sie in den Ratepay <a href='https://www.ratepay.com/legal-payment-dataprivacy/?lang=de' target='blank'>Datenschutzbestimmungen</a> oder nutzen Sie das Ratepay <a href='https://www.ratepay.com/kontakt/' target='blank'>Kontaktformular.</a>"
```

Comment:
We normally ask the translation team to do the language files other than `en`.
Could you please confirm this is necessary?

Reply:
removed translation for de